composer.json

```diff
@@ -45,6 +45,7 @@
         "jms/di-extra-bundle": "~1.7",
         "sensio/generator-bundle": "~2.3|~3.0",
         "symfony/yaml": "~2.3|~3.0",
+        "phpunit/phpunit": "~4.3",
```

Comment:
Hmmm… @core23 @OskarStark do you know why we are using @stable instead of a fixed version. It's great to be stable, but we can still can get BC-breaks.

@mikemeier : you can use this in the meantime I guess.